src/script/addresses.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
To the extent we want a test file like this, it should be an artifact somewhere in the test directory.